pkgs/top-level/all-packages.nix

```diff
@@ -34372,9 +34372,9 @@ with pkgs;
 
   openbrf = libsForQt5.callPackage ../applications/misc/openbrf { };
 
-  opencpn = darwin.apple_sdk_11_0.callPackage ../applications/misc/opencpn {
```

Comment:
Are you sure about this?

Reply:
Fair enough then.

Reply:
> However, I see in the logs that it's not possible to build it on Darwin, have you seen?

[x86_64-darwin](https://logs.ofborg.org/?key=nixos/nixpkgs.281450&attempt_id=b45c4d58-ac8c-451c-84a7-c4a7bce542fb):

```
CMake Error at /nix/store/101zq7jbr3w3q0v2f1xyz9y6d488f4lv-cmake-3.27.8/share/cmake-3.27/Modules/GoogleTestAddTests.cmake:112 (message):
  Error running test executable.
    Path: '/tmp/nix-build-opencpn-5.8.4.drv-0/source/build/test/tests'
    Result: Process terminated due to timeout
    Output:
      
Call Stack (most recent call first):
  /nix/store/101zq7jbr3w3q0v2f1xyz9y6d488f4lv-cmake-3.27.8/share/cmake-3.27/Modules/GoogleTestAddTests.cmake:225 (gtest_discover_tests_impl)
```

[aarch64-darwin](https://logs.ofborg.org/?key=nixos/nixpkgs.281450&attempt_id=d86dc85a-1af8-411b-b88f-fe901e3c840f)

```
CMake Error at /nix/store/slgfvfhi4nbdms9h7p13rp998ls191az-cmake-3.27.8/share/cmake-3.27/Modules/GoogleTestAddTests.cmake:112 (message):
  Error running test executable.
    Path: '/tmp/nix-build-opencpn-5.8.4.drv-0/source/build/test/tests'
    Result: Process terminated due to timeout
    Output:
      
Call Stack (most recent call first):
  /nix/store/slgfvfhi4nbdms9h7p13rp998ls191az-cmake-3.27.8/share/cmake-3.27/Modules/GoogleTestAddTests.cmake:225 (gtest_discover_tests_impl)
```

I ran these both successfully... so I don't know what's up.

Reply:
@NixOS/darwin-maintainers Can you help in here?